frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js

```diff
@@ -885,6 +896,26 @@ export default function lineAreaBar(
 
   checkSeriesIsValid(props);
 
+  if (props.chartType === "waterfall") {
+    props.series.map(series => {
+      const rows = series.data.rows;
+      const finalRow = rows[rows.length - 1];
+      if (!finalRow._waterfallTotal) {
+        const totalValue = rows.reduce((t, d) => (t += d[1]), 0);
```

Comment:
```suggestion
        const totalValue = rows.reduce((t, d) => t + d[1], 0);
```

It doesn't change the behavior, but we don't need to reassign `t`.

Reply:
Oops, good catch!